backend/Tasque.Core.WebAPI/Controllers/UserController.cs

```diff
@@ -47,10 +40,21 @@ public async Task<IActionResult> EditPassword([FromBody] PasswordEditDto dto)
             return Ok(result);
         }
 
+        [HttpGet("current")]
+        public async Task<IActionResult> GetCurrentUserFromToken()
+        {
+            return Ok(await _service.
+                    GetUserById(
+                        GetCurrentUserId()));
+        }
+
         private int GetCurrentUserId()
         {
-            /* Just a stub. Should be implemented */
-            return 1;
+            var id = User.Claims.FirstOrDefault(x => x.Type == "id")?.Value;
+            if (string.IsNullOrEmpty(id))
+                throw new InvalidTokenException("Invalid access token");
+
+            return int.Parse(id);
```

Comment:
Check https://trello.com/c/ImVlCs4P/97-create-services-to-get-userid-from-token for this part

Reply:
> Check https://trello.com/c/ImVlCs4P/97-create-services-to-get-userid-from-token for this part

Done